tests/lighthouse-performance.test.ts

```
@@ -1,36 +1,39 @@
-import { describe, it, expect, beforeAll } from 'vitest';
-import { existsSync } from 'fs';
-import { LighthouseCriteriaValidator, type PerformanceCriteria } from './utils/lighthouse-criteria';
+import { describe, it, expect, beforeAll } from "vitest";
+import { existsSync } from "fs";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)